soc/nordic/common/vpr/Kconfig.sysbuild

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+config VPR_LAUNCHER
+	bool "VPR launcher"
+	depends on (SOC_NRF54H20_CPUPPR || SOC_NRF54L15_ENGA_CPUFLPR)
```

Comment:
Would it be useful to make this `default y`? This will only be activated when the main sysbuild `BOARD` is PPR or FLPR, and not when only including a secondary image for it.

Reply:
This config should only be enabled for VPR targets. Building with it for ARM or single-core targets is not a valid usecase.

Reply:
Yeah, the `depends on` ensures that, so I was suggesting to add `default y` on top of it.

Reply:
Sorry, I misinterpreted this as suggestion to replace `depends on`.
I'd prefer this to be explicitly enabled, so there is less confusion for users that do not know about this functionality.
But either is fine by me.